src/Effect/AVar.js

```diff
@@ -40,18 +39,18 @@ var AVar = function () {
   function putLast (queue, value) {
     var cell = new MutableCell(queue, value);
     switch (queue.size) {
-    case 0:
-      queue.head = cell;
-      break;
-    case 1:
-      cell.prev = queue.head;
-      queue.head.next = cell;
-      queue.last = cell;
-      break;
-    default:
-      cell.prev = queue.last;
-      queue.last.next = cell;
-      queue.last = cell;
+      case 0:
```

Comment:
The eslint configuration requires indents here; there is no semantic change anywhere in this file.

Reply:
I can confirm this fact.